build.gradle

```diff
@@ -57,13 +58,14 @@ dependencies {
 
     implementation 'software.amazon.awssdk:s3'
     implementation 'software.amazon.awssdk:s3-transfer-manager'
-    implementation 'software.amazon.awssdk.crt:aws-crt:0.28.6'
+    implementation 'software.amazon.awssdk.crt:aws-crt:0.28.0'
```

Comment:
Did you mean to downgrade the version here?

Reply:
No 😅 I'll change it back, thanks for the heads up!

Reply:
Done 👍